packages/dart_frog_cli/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.3.1
+
+- feat: upgrade to mason ^0.1.0-dev.38
+  - perf: significant decrease in hook startup times
```

Comment:
Is this indentation intention?

Reply:
Yeah since this is part of the `mason` upgrade.